README.md

```diff
@@ -1,8 +1,5 @@
 # Spring
 
-[![Build Status](https://travis-ci.org/rails/spring.svg?branch=master)](https://travis-ci.org/rails/spring)
-[![Gem Version](https://badge.fury.io/rb/spring.svg)](http://badge.fury.io/rb/spring)
```

Comment:
I asked to move out from the README. For me it doesn't add anything to users since my expectation is that build is always green. In fact I'd just remove it, but moving to contributing.md is fine to me.